packages/lib/global-css/typography.css

```diff
@@ -11,74 +11,165 @@ button {
   font-family: inherit;
 }
 
-.cx-headline-jumbo,
-.cx-headline-1,
-.cx-headline-2,
-.cx-headline-3,
-.cx-headline-4,
-.cx-headline-5 {
+.cx-title-1,
+.cx-title-2,
+.cx-title-3,
+.cx-title-4,
+.cx-title-5 {
   text-wrap: balance;
 }
 
-.cx-headline-jumbo {
-  font-size: 4.375rem;
-  font-weight: 700;
-  font-style: normal;
-  line-height: normal;
-}
-
-.cx-headline-1 {
-  font-weight: 700;
-  font-size: 1.875rem;
-  line-height: 1.33;
+.cx-title-1 {
+  font-family: "Open Sans", Arial, sans-serif;
+  font-weight: 400;
+  font-size: 2.75rem;
+  line-height: 1.6;
 }
 
-.cx-headline-2 {
-  font-weight: 700;
-  font-size: 1.25rem;
-  line-height: 1.5;
+.cx-title-2 {
+  font-family: "Open Sans", Arial, sans-serif;
+  font-weight: 600;
+  font-size: 2.25rem;
+  line-height: 1.6;
 }
 
-.cx-headline-3 {
-  font-weight: 700;
-  font-size: 1rem;
-  line-height: 1.625;
+.cx-title-3 {
+  font-family: "Open Sans", Arial, sans-serif;
+  font-weight: 600;
+  font-size: 1.75rem;
+  line-height: 1.6;
 }
 
-.cx-headline-4 {
-  font-weight: 400;
-  font-size: 1rem;
-  line-height: 1.625;
+.cx-title-4 {
+  font-family: "Open Sans", Arial, sans-serif;
```

Comment:
Er det noen gang vi ikke har   font-family: "Open Sans", Arial, sans-serif;?  For da kunne vi jo egentlig bare definert det på body elementet